bom/application/pom.xml

```diff
@@ -4764,11 +4764,7 @@
                 <groupId>org.jboss.logging</groupId>
                 <artifactId>jboss-logging-annotations</artifactId>
                 <version>${jboss-logging-annotations.version}</version>
-            </dependency>
-            <dependency>
-                <groupId>org.jboss.logging</groupId>
-                <artifactId>jboss-logging-processor</artifactId>
-                <version>${jboss-logging-annotations.version}</version>
+                <scope>provided</scope>
```

Comment:
We usually refrain from adding scopes in the BOM as it makes things less flexible (e.g. if for some reasons, you want to use another scope).

Now I'm not entirely sure using another scope makes sense in this very specific case. WDYT @dmlloyd ?